config/volume.go

```diff
@@ -405,3 +405,17 @@ func (cfg *Volume) ToDriverOptions(timeout time.Duration) (storage.DriverOptions
 func (cfg *Volume) String() string {
 	return path.Join(cfg.PolicyName, cfg.VolumeName)
 }
+
+// IsVolumeInUse checks if the given volume is mounted in any container
+func (c *Client) IsVolumeInUse(cfg *Volume) (bool, error) {
+	if !cfg.Unlocked {
+		if _, err := c.etcdClient.Get(context.Background(), c.use(UseTypeMount, cfg.String()), nil); err != nil {
+			if cErr, ok := err.(client.Error); ok {
```

Comment:
I just did something similar, but my implementation is a little different:  https://github.com/contiv/volplugin/pull/401/commits/b6e5acb5bf31f5b83182336a8dc356842775b361#diff-3fb318b7d8ab9f4eb7fd5d17b99eb42aR44

@erikh: what is the preferred method here?